assets/components/ChatSidebar.svelte

```diff
@@ -11,14 +11,14 @@ import {route} from '../store/Route';
 export let transition;
 
 const user = getContext('user');
-const notifications = $user.notifications;
```

Comment:
Why did you change this? I think `$user.notifications` was a typo in the original code though. It should just be `user.notifications`.

Reply:
Ah I think that one was me trying to fix a different issue, I'll put it back